tests/GeneralStateTestsRunner.js

```diff
@@ -42,6 +44,31 @@ function runTestCase (testData, t, cb) {
       }
 
       if (tx.validate()) {
+        if (options.JSONTrace) {
+          vm.on('step', function (e) {
+            hexStack = []
+            for (var i = 0; i < e.stack.length; i++) {
```

Comment:
Could be using `map` here. `var hexStack = e.stack.map(function (val) { return '0x' + new BN(val).toString(16, 0) })`